Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "GPLikelihoods"
 uuid = "6031954c-0455-49d7-b3b9-3e1c99afaf40"
 authors = ["JuliaGaussianProcesses Team"]
-version = "0.2.7"
+version = "0.3.0"
```

Comment:
Maybe we could make the breaking change of the `GammaLikelihood` in the same release (not sure if the PR is ready)?